packages/react-reconciler/src/ReactFiberExpirationTime.js

```diff
@@ -80,3 +91,31 @@ export function computeInteractiveExpiration(currentTime: ExpirationTime) {
     HIGH_PRIORITY_BATCH_SIZE,
   );
 }
+
+export function inferPriorityFromExpirationTime(
+  currentTime: ExpirationTime,
+  expirationTime: ExpirationTime,
+): ReactPriorityLevel {
```

Comment:
It doesn't matter for "old" because "old" doesn't integrate with Scheduler's priorities regardless.

Reply:
As discussed, this is because for follow up callbacks we need to schedule with a priority. We might get this wrong but only at higher priority. This will cause a problem for priority-based warnings (such as suspending at hi-pri).